app/views/inventories/_inventories_table.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="content">
+  <div class="material-datatables">
+      <table id="inventories-table" class="table table-no-bordered table-hover"> <!--table-striped -->
```

Comment:
The correct placement of an HTML comment it's above the element which was commented. Also, don't use class names in the comments or as storage for classes you don't use.